contracts/StableJoeStaking.sol

```diff
@@ -240,14 +236,14 @@ contract StableJoeStaking is Initializable, OwnableUpgradeable {
     }
 
     /**
-     * @notice Set the Smol Joes address
-     * @param _newSmolJoes The new Smol Joes contract address
+     * @notice Initialize the Smol Joes address
+     * @param _smolJoes The Smol Joes contract address
```

Comment:
Add a comment explaining that this function was added to be able to set smolJoe during an upgrade, as the contract was already initialized